ci/build_container/build_recipes/lightstep.sh

```diff
@@ -7,6 +7,11 @@ VERSION=0.36
 wget -O lightstep-tracer-cpp-$VERSION.tar.gz https://github.com/lightstep/lightstep-tracer-cpp/releases/download/v${VERSION//./_}/lightstep-tracer-cpp-$VERSION.tar.gz
 tar xf lightstep-tracer-cpp-$VERSION.tar.gz
 cd lightstep-tracer-cpp-$VERSION
+
+# see https://github.com/lyft/envoy/issues/1387 for progress
+wget -O ../lightstep-missing-header.diff https://raw.githubusercontent.com/vbatts/copr-build-envoy/2eea2906bfc51a858cf55b894f0ba45f958058e3/lightstep-missing-header.diff
```

Comment:
Could you inline this by a delimited `cat` to a file here, to avoid having to look at GH to know what's happening?